plugins/inputs/sqlserver/README.md

```diff
@@ -66,20 +66,25 @@ GO
 
   # database_type = "AzureSQLDB"
 
-  ## A list of queries to include. If not specified, all the above listed queries are used.
+  ## A list of queries to include. 
   # include_query = []
 
   ## A list of queries to explicitly ignore.
-  # exclude_query = []
+  ## Query Store queries (AzureSQLDBQueryStoreRuntimeStatistics and AzureSQLDBQueryStoreWaitStatistics) are excluded by default.
+  ## If you enable Query Store queries, the collection interval should comply with the following restriction: 15m =< collection interval <= 2h. 
+  ## Intervals shorter than 15m may cause higher performance impact on source. Intervals longer than 2h may cause some Query Store data not to be collected.
+  # exclude_query = ["AzureSQLDBQueryStoreRuntimeStatistics", "AzureSQLDBQueryStoreWaitStatistics"]
```

Comment:
This will not work. You need to set
```toml
# include_query = ["AzureSQLMIResourceStats", "AzureSQLMIResourceGovernance", "AzureSQLMIDatabaseIO", "AzureSQLMIServerProperties", "AzureSQLMIOsWaitstats", "AzureSQLMIMemoryClerks", "AzureSQLMIPerformanceCounters", "AzureSQLMIRequests", "AzureSQLMISchedulers"]
# exclude_query = []
```
to guarantee a consistent behavior if the user uncomments `include_query`. You can still have `exclude_query = ["AzureSQLDBQueryStoreRuntimeStatistics", "AzureSQLDBQueryStoreWaitStatistics"]`, even though it requires the user to modify two settings if he/she wants to enable one of the excluded queries.

Reply:
@srebhan, done.